roles/openshift_project/tasks/delete-project.yaml

```diff
@@ -0,0 +1,27 @@
+- name: "{{ project_name }} : delete namespace"
+  kubernetes.core.k8s:
+    state: absent
+    definition:
+      apiVersion: v1
+      kind: Namespace
+      metadata:
+        name: "{{ project_name }}"
+        namespace: "{{ project_name }}"
+
+- name: "{{ project_name }} : delete admin group"
+  kubernetes.core.k8s:
+    state: absent
+    definition:
+      apiVersion: user.openshift.io/v1
+      kind: Group
+      metadata:
+        name: "{{ project_name }}-admin"
+
+- name: "{{ project_name }} : delete member group"
+  kubernetes.core.k8s:
+    state: absent
+    definition:
+      apiVersion: user.openshift.io/v1
+      kind: Group
+      metadata:
+        name: "{{ project_name }}-member"
```

Comment:
I guess kubernetes doesn't force you to remove the rolebindings first without removing these groups.

Reply:
RIght, because there's not a hard dependency between the two: there could be multiple rolebindings associated with the group, or zero. E.g., maybe you're just using the group for membership in an app or something. Also, we *are* deleting the rolebindings beforehand because we delete the namespace first, which deletes everything inside the namespace (including the rolebindings).